src/client/src/system/service/autobahnConnectionProxy.js

```diff
@@ -9,11 +9,23 @@ export default class AutobahnConnectionProxy {
   connection:autobahn.Connection;
 
   constructor(url:string, realm:string) {
+    const useSecure = location.protocol === 'https:';
+    const securePort = 443;
+    const defaultPort = 8080;
     this.connection = new autobahn.Connection({
-      url: url,
       realm: realm,
       use_es6_promises: true,
-      max_retries: -1 // unlimited retries
+      max_retries: -1, // unlimited retries,
+      transports: [
```

Comment:
Bit surprised to only see 2 options here. Were there more than 2 options in the default list?